Library/Homebrew/cmd/doctor.rb

```diff
@@ -92,6 +92,28 @@ def check_for_macgpg2
     end
   end
 
+  # Anaconda installs multiple system & brew dupes, including OpenSSL, Python,
+  # sqlite, libpng, Qt, etc. Regularly breaks compile on Vim, MacVim and others.
+  # Is flagged as part of the *-config script checks below, but people seem
+  # to ignore those as warnings rather than extremely likely breakage.
+  def check_for_anaconda
+    return unless which("anaconda")
+    anaconda = which("anaconda").dirname.realpath
+    python = which("python").dirname.realpath
+
+    # Only warn if Python lives with Anaconda, since is most problematic case.
+    if python == anaconda then <<-EOS.undent
+      Anaconda is known to frequently break Homebrew builds, including Vim and
+      MacVim, due to bundling many duplicates of system and Homebrew-available
+      tools.
+
+      If you encounter build failure with Anaconda installed please temporarily
+      remove it from your $PATH and attempt the build again prior to reporting
+      the failure to us. Thanks!
```

Comment:
Nits for clarity, feel free to ignore: "If you encounter __a__ build failure"; "remove __Anaconda__ from your PATH"